app/hooks/useEvents.ts

```diff
@@ -0,0 +1,9 @@
+import type { SerializeFrom } from '@remix-run/node';
+import { useRouteLoaderData } from '@remix-run/react';
+import type { loader } from '~/root';
+
+export function useEvents() {
```

Comment:
As mentioned during the pair coding session, loading the events in the root `loader` and making hooks may not be necessary if we're only going to be using these events in the `/events` route. The events can be loaded in the route's `loader` function instead. This could potentially help with the loading speed of our homepage. Let's leave things as-is for now and refactor if needed in the future :)

Reply:
I don't know how complex our web app will be :-) If these hooks are reusable enough, we might consider using them for the "Upcoming Events" section or on the "Search" page.